.github/actions/restore-node/action.yml

```diff
@@ -114,6 +114,8 @@ runs:
           ${{ inputs.path }}/endo-sha.txt
     - uses: kenchan0130/actions-system-info@master
       id: system-info
+    - run: corepack enable
+      shell: bash
```

Comment:
> I think we can remove the latter.

Done.

> Also I don't think the shell property is necessary.

I've found that `shell: bash` is necessary for every `run: xxx` step because `restore-node/action.yml` is a [composite action](https://docs.github.com/en/actions/creating-actions/creating-a-composite-action#creating-an-action-metadata-file), not a workflow.  See the Github documentation that [describes `shell:`](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#runsstepsshell) in the [composite actions metadata syntax section](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#runs-for-composite-actions).